andela/thorin-rc
# #157906020 Embed social media content

### What does this PR do?

Embed social media content in the application

### Description of tasks to be completed

- Create tweet collection with hashtag #ShopThorinRC
- Embed tweet collection in the footer
- Enable social media sharing for products
- Enable social media login

### How should this be manually tested ?
Clone the repository and run `reaction`. Open the browser and go to `127.0.0.1:3000`, you should see the twitter embed in the footer.
Social media login and social media share for products can be tested on the app's staging [url](https://thorin-rc-staging.herokuapp.com/). Navigate to the landing page and you should option to sign in and signup using Facebook and Google. Navigate to a product's details page and you should see the option to share a product on Facebook and Twitter.

### Any background context you want to provide ?
N/A

### Relevant screenshots
<img width="1440" alt="screen shot 2018-06-26 at 7 32 30 am" src="https://user-images.githubusercontent.com/25195857/41893828-e88a1226-7914-11e8-9008-361d5b77f456.png">


### Any relevant pivotal tracker stories ?
#157906020

### Questions
N?A
